test/language/class/class-strict-mode.js

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) Copyright 2014 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 14.5
+description: >
+    class strict mode
```

Comment:
Nevermind, wrong interpretation. In fact maybe it should be tagged noStrict, given that it has a top-level `with`.